src/components/CalendarMonthGrid.jsx

```diff
@@ -78,6 +79,7 @@ const defaultProps = {
   firstDayOfWeek: null,
   setCalendarMonthHeights() {},
   isRTL: false,
+  transitionDurationMS: 200,
```

Comment:
i don't know if the "MS" prefix is actually helpful here; durations in JS are almost always in milliseconds already (like setTimeout)

Reply:
Well except for the fact that we were previously doing the transition in seconds... I think it is still useful metadata to have in the prop name.

Reply:
previously, we weren't accepting a transition at all, so nobody knew the unit. That in a previous iteration of the PR it took seconds isn't relevant.

Reply:
Good call; it should be documented regardless.